packages/@webex/plugin-meetings/test/unit/spec/meeting/voicea-meeting.ts

```diff
@@ -253,5 +254,51 @@ describe('plugin-meetings', () => {
                 expect(newCaption.speaker).to.deep.equal(speaker);
             });
         });
+
+        describe('processHighlightCreated', () => {
+            beforeEach(() => {
+              fakeVoiceaPayload = {
```

Comment:
Here too - had to mock a lot of values in order to properly unit test the required method.